### DISCUSSION TITLE
Added: Notes for November 11th, 2020

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4128] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-11-11.md

## Agenda

- IsRecord in metadata
- Triage

**Comment by bartdesmet** [comment]
It seems like the link for the `AsyncMethodBuilder` section is invalid.

**Reply by 333fred** [reply]
Thanks, fixed.
